src/Jab/Attributes.cs

```diff
@@ -1,5 +1,7 @@
 ﻿// <auto-generated/>
 
+#if !JAB_ATTRIBUTES_REFERENCED
```

Comment:
Do you mind linking both projects to the same Attributes.cs file with `#if`s that control type accessibility? 

Similar to how .NET runtime does [here](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/Diagnostics/CodeAnalysis/NullableAttributes.cs#L9-L13)

Reply:
No problem. Do you have preference where the attributes file should be located?

Reply:
Done, kept the one on Jab project